src/behat/Api/Context/RestContextTrait.php

```diff
@@ -504,4 +504,12 @@ public function theResponseCodeShouldBe(int $expectedCode)
             . $this->getHttpResponse()->getBody()->__toString();
         Assert::eq($expectedCode, $actualCode, $message);
     }
+
+    /**
+     * @Given I Wait until broker bulk is done
+     */
+    public function iWaitUntilBrokerBulkIsDone()
+    {
+        sleep(10);
```

Comment:
Why not taking a small margin of 100ms ?
```suggestion
        usleep(10_100_000); // 10.1 seconds
```